### DISCUSSION TITLE
Новая модель V380E13

### DISCUSSION THREAD

**Discussion by alaponicin** [Arkady23/V380-Web#7] [Ideas]
Здравствуйте! Добавьте пожалуйста новую модель:
Software version: AppV380E13_PTZ_V2.5.10.6_20190807
Firmware version: HwV380E13_WF3_PTZ_20160217
Заранее благодарен!

**Comment by Arkady23** [comment]
Я добавил. Но вы не сообщили Hwname. Вы должны его вытащить из камеры. А иначе я не смогу установщик сделать.

**Reply by alaponicin** [reply]
Не сразу разобрался, как его вытащить :)
Hwname : V380E2_CA

**Reply by Arkady23** [reply]
Кстати, ваш установщик есть в проекте, просто его не было в таблице. В таблицу добавил. Прямая ссылка https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E2_CA_v20210612.zip Теперь все, у кого такая камера, могут воспользоваться информацией о том какое у этой камеры hwname.
Я столкнулся с ситуацией, что в некоторых камерах другой Linux и web-сервер на запускается. Сейчас разбираюсь, как сделать, чтоб работал на этом Linux. Если получится, то завтра или на этой неделе буду учитывать у кого какой Linux. Какаой у вас не известно. Попробуйте, потом сообщите.

**Reply by Arkady23** [reply]
У вас установился web-интерфейс? Если нет, то устанавливайте! Патчи обновились.

**Comment by hamx71** [comment]
Tienes para esta amigo . 
<img width="1080" height="2400" alt="Screenshot_2026-05-12-20-58-39-026_com macrovideo v380pro" src="https://github.com/user-attachments/assets/816dc8f9-39cd-48df-9baa-e7c35af7a8f1" />
